addon/styles/_frost-sidebar.scss

```diff
@@ -1,62 +1,52 @@
-.frost-sidebar .container {
-  width: 100%;
-  height: 100%;
-}
-
 .frost-sidebar {
   position: absolute;
-  left: 0;
+  left: 0px;
+
+  .liquid-container > .liquid-child {
+    width: auto;
+    height: auto;
+  }
+
+  .open-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```

Comment:
You mean we should leave it to the caller how they want to lay out their content within the content container. I was thinking we should have one default and then caller can override it if they want. This one gives the lazy caller some default layout. No?